### PR TITLE
Use fluentd v0.12 and td-agent 2.2.0

### PR DESCRIPTION
**Pull request by repeatedly**

### CODE REVIEW

**Comment by kenjiskywalker** [comment]
awesome :innocent:

**Comment by repeatedly** [comment]
Just fixed two regression. Time to release td-agent 2.2.0.

**Comment by repeatedly** [comment]
I will try to test ruby 2.2.0 to check performance improvement.
After that, I will release td-agent 2.2.0.

**Comment by kenjiskywalker** [comment]
Great! :pray:

File: config/software/ruby.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@
 #
 
 name "ruby"
-default_version "2.1.3"
+default_version "1.9.3-p550"
```
Comment (kenjiskywalker) [inline config/software/ruby.rb]: @repeatedly 

Why down grade `default_version` ?

Comment (repeatedly) [inline config/software/ruby.rb]: See this commit: https://github.com/treasure-data/omnibus-td-agent/commit/e3d1b107d081b9d290950b72a6c3762573174d0e

Downgrade to default_version is for easy to update ruby.rb.
omnibus-software's ruby is still use 1.9.3 as a default version.

Comment (kenjiskywalker) [inline config/software/ruby.rb]: > e3d1b10

oh sorry :bow: 

> omnibus-software's ruby is still use 1.9.3 as a default version.

Hmmm. Thanks :smile:

**Comment by repeatedly** [comment]
I tested ruby 2.2.1 locally.
Memory usage is stable on 2.1.5. 2.2.1 needs a bit of more memory periodically.
I think incremental GC causes this behaviour.
I will ask some people about this behaviour is problem or not.

**Pull request merged**